packages/expo-constants/scripts/with-node.sh

```diff
@@ -40,5 +40,6 @@ fi
 
 # Execute argument, if present
 if [[ "$#" -gt 0 ]]; then
-  "$NODE_BINARY" $@
+  args=( "$@" )
+  "$NODE_BINARY" "${args[@]}"
```

Comment:
Typically `"$@"` is used to pass through all of the arguments. Does that work here?
```suggestion
  "$NODE_BINARY" "$@"
```

Reply:
cool it works! good to know this trick, thanks James!